chain/consensus/mir/manager.go

```diff
@@ -92,11 +93,16 @@ func NewManager(ctx context.Context,
 	}
 	id := cfg.Addr.String()
 
+	ts, err := node.ChainHead(ctx)
```

Comment:
This is not correct, here you are using a tipset from the child to make a call in the parent. I think we can use the empty tipset key directly, I'll fix it myself